host/sgx/calls.c

```diff
@@ -390,6 +431,19 @@ static oe_result_t _handle_ocall(
     if (arg_out)
         *arg_out = 0;
 
+    if (func != OE_OCALL_LOG)
```

Comment:
Why don't we log OE_OCALL_LOG calls?

Reply:
I'm assuming endless recursive loop. You shouldn't log something in your log function.

Reply:
There is no recursive loop here because this is on the host side, it won’t cause another OCall.
It’s two lines for each log line though, it seemed a bit excessive, but I don’t feel strongly either way, I am happy to put it back if we think it’s preferable.

Reply:
@jxyang how do you feel about this? I'm happy either way.

Reply:
We should cover OE_OCALL_LOG as well for the sake of completeness. Like @achamayou said, there is no risk of recursive loop.

Reply:
Done